src/main/java/bio/terra/service/dataset/DatasetDao.java

```diff
@@ -515,4 +515,20 @@ public DatasetSummary mapRow(ResultSet rs, int rowNum) throws SQLException {
                 .createdDate(rs.getTimestamp("created_date").toInstant());
         }
     }
+
+    /**
+     * Probe to see if can access database
+     */
+    public boolean statusCheck() {
+        String sql = "Select count(1)";
```

Comment:
```suggestion
        String sql = "SELECT now()";
```

It might be useful to return a timestamp for debugging! Then you can log when it succeeded too.

Reply:
I updated "Select" to "SELECT."
RE: returning now(): Since the /status endpoint is hit so frequently, it would be too noisy to log on success. So, I think the simple "count(1)" return is sufficient.

Reply:
Thank you for giving it a shot!